src/libraries/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.Shared.xml

```diff
@@ -141,6 +141,9 @@
     <type fullname="System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAttribute">
       <attribute internal="RemoveAttributeInstances" />
     </type>
+    <type fullname="System.Diagnostics.CodeAnalysis.ExperimentalAttribute">
+      <attribute internal="RemoveAttributeInstances" />
+    </type>
```

Comment:
I _think_ that the attribute is no longer relevant at the point an application is being trimmed. But I don't claim I understand this area too well.

Reply:
Yes, that's fine.